src/qt/transactionview.cpp

```diff
@@ -592,6 +591,27 @@ void TransactionView::focusTransaction(const QModelIndex &idx)
     transactionView->setFocus();
 }
 
+void TransactionView::focusTransaction(const uint256& txid)
+{
+    if (!transactionProxyModel)
+        return;
+
+    const QModelIndexList results = this->model->getTransactionTableModel()->match(
+        this->model->getTransactionTableModel()->index(0,0),
+        TransactionTableModel::TxHashRole,
+        QString::fromStdString(txid.ToString()), -1);
+
+    transactionView->setFocus();
+    transactionView->selectionModel()->clearSelection();
+    for (const QModelIndex& index : results) {
+        const QModelIndex targetIndex = transactionProxyModel->mapFromSource(index);
+        transactionView->selectionModel()->select(
+            targetIndex,
+            QItemSelectionModel::Rows | QItemSelectionModel::Select);
+        if (index == results[0]) transactionView->scrollTo(targetIndex);
```

Comment:
On my system calling `transactionView->scrollTo(targetIndex)` for every model index fixes it (not sure why). It also works in the default order because scroll hint is `QAbstractItemView::EnsureVisible`, so focusing the 2nd row will not remove the 1st from the viewport (the 2nd row is already visible).

But it's also possible to order by other column (for instance address), and in that case it can be impossible to show the whole selection in the table viewport.

I see 3 possible solutions for that:
 - do nothing :trollface:;
 - automatically sort by date descending (new rows will be the first);
 - automatically fill the filter field with the new txid.

Reply:
When I call `scrollTo` for every model index and transactions are sort by ascending date, things get weird: if I send to two destinations it scrolls correctly, but if I send to one destination it won't scroll far enough.

Calling `scollTo()` twice "fixes" it. Seems suspiciously similar to [this](https://forum.qt.io/topic/81671/scrollto-not-working-properly/4), I'll poke around a bit..

I prefer your first solution for the case where transactions are not ordered by date. At least one of them will be in view.